jerry-core/lit/lit-strings.c

```diff
@@ -17,6 +17,15 @@
 
 #include "jrt-libc-includes.h"
 
+#define LIT_UTF8_SURROGATE_MARKER (0xed)
+#define LIT_UTF8_HIGH_SURROGATE_MIN (0xa0)
+#define LIT_UTF8_HIGH_SURROGATE_MAX (0xaf)
+#define LIT_UTF8_LOW_SURROGATE_MIN (0xb0)
+#define LIT_UTF8_LOW_SURROGATE_MAX (0xbf)
+#define LIT_UTF8_1_BYTE_MAX (0xf4)
+#define LIT_UTF8_2_BYTE_MAX (0x8f)
+#define LIT_UTF8_OVERLONG_MARKER (0xc2)
```

Comment:
LIT_UTF8_OVERLONG_MARKER -> LIT_UTF8_VALID_TWO_BYTE_START
There are no comments for the constants